src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/emails/court/CourtEmailFactory.kt

```diff
@@ -14,6 +14,8 @@ import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.Prisoner
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.BookingAction
 import java.util.UUID
 
+const val DEFAULT_COURT_URL_SUFFIX = "meet.video.justice.gov.uk"
+
```

Comment:
I've got a feeling this might become a pattern to format and inject known values, if present. But fine for now.